### ISSUE TITLE
rviz in cyber

### ISSUE THREAD

**Issue by sinopec** [ApolloAuto/apollo#14896]
currently the cyber_visulizer is really ugly  in functionality richness and stability.
Is therer a plan to improve this tool to make it more useful? 
Or any plan to  offer a official bridge to convert cyber mesasge to ros message ? 
It will be really helpful for develoment.

**Comment by daohu527** [comment]
yeah, you're right, for now apollo have two visualization solutions
- dreamview
- cyber_visulizer

The good news is that we are optimizing dreamview, the main problem with enhancing cyber_visulizer is that
- apollo for now run in docker, visualization in docker had some problems before, I didn't investigate carefully, of course I think we can optimize it
- Fully benchmarking rviz needs to be redesigned, which may take a while

What suggestions do you have? We're also happy to provide better visualization tools

**Comment by sinopec** [comment]
> 

Maybe apollo can provide a official apollo_ros_bridge  first . I know someone has provide a project , but honesly It doesnt work very well , speciallly when I have multi  high freq channle to convert.
